config/network-policy/allow-metrics-traffic.yaml

```diff
@@ -0,0 +1,27 @@
+# This NetworkPolicy allows ingress traffic
```

Comment:
Why is this in a kustomization and not in the helm chart ?

Reply:
Everything in the configs folder is generated by kube-builder :)
I mostly applied the one generated by the latest kube-builder